### PR TITLE
Adding support for slack

### PR DESCRIPTION
**Pull request by TehreemNisa**

### CODE REVIEW

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -54,6 +54,17 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - slack.stakater.com/v1alpha1
```
Comment (bnallapeta) [inline config/rbac/role.yaml:58]: this is defined twice in this file. Either clean up this one or the one defined at the end.

File: roles/grafana_cloud_operator/tasks/grafana_oncall_hub_spoke.yml [hunk 2]
```diff
@@ -36,6 +36,28 @@
...
+        label: "{{ item.namespace }}"
+
+    - name: Extract Channel id from slack channel
+      ansible.builtin.set_fact:
+        slack_channel_ids: "{{ slack_channel_ids | default([]) + [{'name': item.metadata.name, 'slack_id': item.status.id}] }}"
+      loop: "{{ slack_channel_info.results | map(attribute='resources') | flatten }}"
+      loop_control:
+        label: "{{ item.metadata.name }}"
+
+    - name: Print the slack id and name for each channel
```
Comment (bnallapeta) [inline roles/grafana_cloud_operator/tasks/grafana_oncall_hub_spoke.yml:56]: I suppose this is used only for debug. Remove this step if its not required.

File: config/rbac/role.yaml (hunk 1 above)
Comment (TehreemNisa) [reply config/rbac/role.yaml:58]: I am testing , current code is not working ... once it does will remove the extra one

Comment (bnallapeta) [reply config/rbac/role.yaml:58]: ok, the PR is up for merge yet or no? 
if yes, please test it out with which one works and then remove the other before merging it please.

File: roles/grafana_cloud_operator/tasks/grafana_oncall_hub_spoke.yml (hunk 2 above)
Comment (TehreemNisa) [reply roles/grafana_cloud_operator/tasks/grafana_oncall_hub_spoke.yml]: removed

File: config/rbac/role.yaml (hunk 1 above)
Comment (TehreemNisa) [reply config/rbac/role.yaml]: done

**Review by karl-johan-grahn — APPROVED** [review]

**Review by bnallapeta — APPROVED** [review]

**Pull request merged**